### DISCUSSION TITLE
My code is  executing but the  html tags get lost

### DISCUSSION THREAD

**Discussion by radha1990kumari** [SilentVoid13/Templater#348] [Help] [answered]
```js
<center> <h1>Lab Assignment 
<%*
let re = new RegExp('Lab-(\\d+)_.*');
var myString = tp.file.title;
var match = re.exec(myString); 
var an="0"
if(match!=null)
an=match[1]
tR=an
%>
 </h1>
</center>
```

**Comment by welpdx** [comment, marked as answer]
That is because `tR` is the output of the script.
When you directly assign `an` to `tR`, the resulting output is only `an` and nothing else. It seemed like you lost your html tags, but what actually happened is you basically told the script to output only `an` when you did `tR=an`. You would lose any text before and after any script if you used `tR=an` anywhere in a Templater script. 

So, **what you want to do is instead is to replace the line** `tR=an` with `tR+=an` . This way `an` is inserted between the text you added before and after.

Hope that helps,
Welp

**Reply by radha1990kumari** [reply]
Thanks it works

**Reply by welpdx** [reply]
Np
